test/object-store/bson.cpp

```diff
@@ -200,7 +200,7 @@ TEST_CASE("canonical_extjson_corpus", "[bson]")
 
     SECTION("Double type")
     {
-        static float epsilon = 0.000000001;
+        static float epsilon = 0.000000001f;
```

Comment:
This should probably just be a double since all math is using doubles. Also, it should probably be `static const`.